components/app.js

```diff
@@ -0,0 +1,27 @@
+class App {
+  constructor(gradeTable) {
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+    this.gradeTable = gradeTable;
+  }
+  handleGetGradesError(error){
+    console.error("error: ", error);
+  }
+  handleGetGradesSuccess(grades){
+    this.gradeTable.updateGrades(grades);
+  }
+
+  getGrades(){
+    $.ajax({
+      type:"GET",
+      url:"https://sgt.lfzprototypes.com/api/grades",
+      data:"none",
```

Comment:
`data; "none"` sends a request body containing `none`. If you don't wish to send a request body, don't supply a `.data` property.